provider/resources.go

```diff
@@ -7179,16 +7179,20 @@ $ pulumi import aws:networkfirewall/resourcePolicy:ResourcePolicy example arn:aw
 		if _, ok := fields["tags_all"]; !ok {
 			fields["tags_all"] = &tfbridge.SchemaInfo{}
 		}
+
 		fields["tags_all"].Secret = &yes
 		fields["tags_all"].DeprecationMessage = "Please use `tags` instead."
 
+		// Upstream provider is edited to unmark tags_all as computed internally so that
+		// Pulumi provider internals can set it, but the user should not be able to set it.
+		fields["tags_all"].MarkAsComputedOnly = &yes
```

Comment:
TIL about `tfbridge.True()` that avoids having to construct a bool var just so we can reference it.

Reply:
Most of my go code ends up with a `ref` function:

```go
func ref[T any](t T) *T { return &t }
```